README.md

```diff
@@ -25,6 +25,8 @@ Feel free to add your own project(s)—just fork and make a pull request! We sug
 - **[modrinth/code/apps/frontend](https://github.com/modrinth/code/tree/main/apps/frontend)** ([Website](https://modrinth.com)) - The official Modrinth frontend we're all familiar with
 - **[TheClashFruit/Rithle](https://github.com/TheClashFruit/Rithle)** - An Android app for Modrinth written in Kotlin
 
+- **[onlive1337/ModForge AI](https://github.com/onlive1337/ModForge-Ai)** - ModForge AI is an intelligent Minecraft modpack generator that uses AI and Modrinth api to create customized modpacks based on user descriptions.
```

Comment:
Please organize this alphabetically, capitalize "API", remove the full stop at the end, change the wording to be more neutral (i.e. remove "intelligent"), and don't restate the app name (i.e. just put "A Minecraft modpack generator...")